cura/Settings/CuraContainerStack.py

```diff
@@ -87,6 +89,19 @@ def setQualityChanges(self, new_quality_changes: InstanceContainer, postpone_emi
     def qualityChanges(self) -> InstanceContainer:
         return cast(InstanceContainer, self._containers[_ContainerIndexes.QualityChanges])
 
+    ##  Set the intent container.
+    #
+    #   \param new_intent The new intent container. It is expected to have a "type" metadata entry with the value "intent".
+    def setIntent(self, new_intent: InstanceContainer, postpone_emit: bool = False) -> None:
```

Comment:
```suggestion
    def setIntent(self, new_intent: InstanceContainer, *, postpone_emit: bool = False) -> None:
```
We're not very consistent with this, but we should try to prevent bool traps. By adding the * to it, you prevent this function from being used as setIntent(self, new_intent, True) (instead forcing to use setIntent(new_intent, postpone_emit = True))